_shared/project/.github/workflows/ci.yml

```diff
@@ -92,9 +92,9 @@ jobs:
       {% endif %}
       {% if job in ["tests", "functests"] and cookiecutter.get("postgres") == "yes" %}
       - name: Create test database
-        run: psql -U postgres -h localhost -p {{ cookiecutter['__postgres_port'] }} -c 'CREATE DATABASE {{ cookiecutter.package_name }}_test'
+        run: psql -U postgres -h localhost -p {{ cookiecutter['__postgres_port'] }} -c 'CREATE DATABASE {{ cookiecutter.package_name }}_{{ job }}'
       {% endif %}
-      {% if cookiecutter.get("frontend") == "yes" %}
+      {% if cookiecutter.get("frontend") == "yes" and job in ["tests", "functests"] %}
```

Comment:
Only install the frontend dependencies for the tests and functests CI jobs (not for formatting, linting, coverage).